SU2_CFD/src/solver_direct_turbulent.cpp

```diff
@@ -1176,8 +1173,6 @@ void CTurbSASolver::Preprocessing(CGeometry *geometry, CSolver **solver_containe
 
   if (limiter_turb) SetSolution_Limiter(geometry, config);
 
-  if (limiter_flow) solver_container[FLOW_SOL]->SetPrimitive_Limiter(geometry, config);
-
```

Comment:
Person familiar with the SA and SST solvers, do we need this call to `SetPrimitive_Limiter`? There is already one in the preprocessing method of the Euler/NS solvers.

Reply:
I can remember that we had that discussion already a long time ago. Cannot remember what the conclusion was though. In my opinion it should not matter, whether we use the updated values or not ... and apparently it does not since the regression tests pass.

Reply:
I am not sure they would be updated, since the gradients are not. Let's give it a couple days before merging just in case. Thanks for the review.

Reply:
Yep, we have had this discussion before. I am also fine with removing them (in the typical case with upwind for mean flow and turb, everything will converge to the same solution anyway), and it will give a little performance boost.

I recall that the sticking point was whether any inconsistencies between the numerical methods in the mean flow and turb models would result in some problems, e.g., mixing JST for the mean flow with a second-order upwind method for the turb scalars (which uses the primitive limiter on the mean flow state too in CTurb:UpwindResid()). We should clarify this finally, even if it just means adding some error check.

Reply:
Oh... I see what you mean, that means the turbulence models are being solved for a velocity field that is not divergence-free. I think this warrants further discussion, is that old discussion in an issue? (all I could find was #422). If not I will open one.

Reply:
I think I found the old discussion on this in #403

Reply:
Thank you, I'll have a look.